common/src/main/java/com/scottlogic/deg/common/profile/HelixDateTime.java

```diff
@@ -69,4 +63,23 @@ private static void validateDateRange(OffsetDateTime dateTime)
         if (dateTime != null && dateTime.getYear() <= 9999 && dateTime.getYear() >= 1) return;
         throw new ValidationException("Dates must be between 0001-01-01T00:00:00.000Z and 9999-12-31T23:59:59.999Z");
     }
+
+    private static DateTimeFormatter getDateTimeFormatter() {
+        DateTimeFormatter dateFormat = new DateTimeFormatterBuilder()
+            .appendPattern("u-MM-dd")
+            .parseDefaulting(ChronoField.SECOND_OF_DAY,0)
+            .toFormatter();
+
+        DateTimeFormatter dateTimeFormatter = new DateTimeFormatterBuilder()
+            .append(DateTimeFormatter.ofPattern("u-MM-dd'T'HH:mm:ss'.'SSS"))
+            .optionalStart()
+            .appendOffset("+HH", "Z")
+            .toFormatter();
+
+        return new DateTimeFormatterBuilder()
```

Comment:
nice, so when we come to making dates easier, we'll need to expand this method to accept lots of styles of dates